models/git/lfs_lock.go

```diff
@@ -186,6 +186,20 @@ func CheckLFSAccessForRepo(ctx context.Context, ownerID int64, repo *repo_model.
 	if ownerID == 0 {
 		return ErrLFSUnauthorizedAction{repo.ID, "undefined", mode}
 	}
+	if ownerID == user_model.ActionsUserID {
+		taskId, ok := ctx.Value(access_model.ActionsTaskIDKey).(int64)
```

Comment:
Here I get the taskID from the lfs git api.

For lfs settings taskID is never present and the actions user can not authenticate. Test for lfs cli included now.